docs/sphinx/developer/arch/supported/riscv.rst

```diff
@@ -10,16 +10,39 @@ Decoding
 
 Instruction decoding is performed using the `Capstone <https://github.com/aquynh/capstone/>`_ disassembly framework. The disassembly generated by Capstone is used to determine the properties, operands, and execution behaviour of the corresponding instruction.
 
-The logic held in ``src/lib/arch/riscv/Instruction_decode.cc`` is primarily associated with converting the provided Capstone instruction metadata into the appropriate SimEng ``instruction`` format. Additionally, an instruction's identifiers are defined here through operand usage and opcode values. For the RISCV architecture model, the following identifiers are defined:
+The logic held in ``src/lib/arch/riscv/Instruction_decode.cc`` is primarily associated with converting the provided Capstone instruction metadata into the appropriate SimEng ``instruction`` format. Additionally, an instruction's identifiers are defined here through operand usage and opcode values. For the RISC-V architecture model, the following identifiers are defined:
 
-- ``isLoad_``, is a load operation.
 - ``isStore_``, is a store operation.
+- ``isLoad_``, is a load operation.
+- ``isBranch_``, is a branch operation.
+- ``isMultiply_``, is a multiply operation.
+- ``isDivide_``, is a divide operation.
+- ``isShift_``, is a shift operation.
 - ``isAtomic_``, is an atomic operation.
+- ``isLogical_``, is a logical operation e.g bitwise and.
+- ``isCompare_``, is a compare operation.
+
+.. _riscv-instruction-groups:
+
+Instruction Groups
+******************
+Through a combination of the above identifiers, an instruction can be allocated an :ref:`instruction group <instruction-group>`. The instruction groups available to the RISC-V ISA are detailed below:
+
+.. image:: ../../../assets/instruction_groups_RISCV.png
+  :alt: RISC-V instruction groups
+
+The above diagram follows the same structure as :ref:`AArch64 instruction groups <aarch64-instruction-groups>`. Each level of the diagram represents a different scope of instructions supported, the primary/top-level encapsulates the most instructions whilst the tertiary/bottom-level the least. Each of the above levels is combined through the _ character, working from the top level of the diagram to the bottom. Lower levels are not required if a larger set of instructions is desired. For example the instruction group INT is valid and would encapsulate all instructions that perform integer operations, while INT_MUL is also valid but would only encapsulate the 5 multiply instructions.
```

Comment:
Look at the AArch64 equivalent paragraph and copy the formatting re `` `` usage (e.g. for "_", "INT_MUL", "SimEng/build/_deps/capstone-lib-src/arch/RISCV/RISCVGenInstrInfo.inc", etc)